prusti-tests/tests/compiletest.rs

```diff
@@ -11,17 +11,31 @@ use prusti_server::ServerSideService;
 use std::{env, path::PathBuf};
 
 fn get_prusti_rustc_path() -> PathBuf {
-    let local_prusti_rustc_path: PathBuf = if cfg!(windows) {
-        ["target", "debug", "prusti-rustc.exe"].iter().collect()
+    let local_prusti_rustc_path: PathBuf = if cfg!(debug_assertions) {
+        if cfg!(windows) {
+            ["target", "debug", "prusti-rustc.exe"].iter().collect()
+        } else {
+            ["target", "debug", "prusti-rustc"].iter().collect()
+        }
     } else {
-        ["target", "debug", "prusti-rustc"].iter().collect()
+        if cfg!(windows) {
+            ["target", "release", "prusti-rustc.exe"].iter().collect()
+        } else {
+            ["target", "release", "prusti-rustc"].iter().collect()
+        }
     };
-    let workspace_prusti_rustc_path: PathBuf = if cfg!(windows) {
-        ["..", "target", "debug", "prusti-rustc.exe"]
-            .iter()
-            .collect()
+    let workspace_prusti_rustc_path: PathBuf = if cfg!(debug_assertions) {
+        if cfg!(windows) {
+            ["..", "target", "debug", "prusti-rustc.exe"].iter().collect()
+        } else {
+            ["..", "target", "debug", "prusti-rustc"].iter().collect()
+        }
     } else {
-        ["..", "target", "debug", "prusti-rustc"].iter().collect()
+        if cfg!(windows) {
+            ["..", "target", "release", "prusti-rustc.exe"].iter().collect()
+        } else {
+            ["..", "target", "release", "prusti-rustc"].iter().collect()
+        }
     };
```

Comment:
```rust
let target_directory = ...;
let executable_name = ...;
let local_prusti_rustc_path: PathBuf = ["target", target_directory, executable_name].iter().collect();
let workspace_prusti_rustc_path: PathBuf = ["..", "target", target_directory, executable_name].iter().collect();
```

Reply:
Feel free to push the change; I'm going to lunch :smile: